src/main/java/mod/gate/events/ActionbarReceiveEvent.java

```diff
@@ -0,0 +1,34 @@
+package mod.gate.events;
+
+import net.minecraft.text.Text;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import java.util.regex.Pattern;
+
+public class ActionbarReceiveEvent {
+    //Convenience Fields
+    public String content;
+    public int timeRemaining;
+
+    //Patterns
```

Comment:
Put the pattern at the top since it's final, followed by required then convenience. Looks good though, I'll change the regex myself to have capture groups for data